bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -1816,25 +1816,7 @@
 	<channel-type id="temperature-channel">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature</label>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="feels-like-channel">
-		<item-type>Number:Temperature</item-type>
-		<label>Feels Like</label>
-		<description>what the temperature feels like accounting for heat index or wind chill. Daily values are average values
-			(mean) for the day.</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="humidity-channel">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Humidity</label>
-		<description>relative humidity in %</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="dew-channel">
-		<item-type>Number:Temperature</item-type>
-		<label>Dew</label>
-		<description>dew point temperature</description>
+		<category>Temperature</category>
 		<state readOnly="true"/>
```

Comment:
I guess you would also want to add a %unit% state pattern?

Reply:
why?

Reply:
Because this will format the value and unit independent to the user locale (si vs imperial) etc.)

Reply:
done